tests/test_log_command.rs

```diff
@@ -288,6 +288,43 @@ fn test_log_with_or_without_diff() {
     "###);
 }
 
+#[test]
+fn test_log_divergence() {
+    let test_env = TestEnvironment::default();
+    test_env.jj_cmd_success(test_env.env_root(), &["init", "repo", "--git"]);
+    let repo_path = test_env.env_root().join("repo");
+
+    std::fs::write(repo_path.join("file"), "foo\n").unwrap();
+    test_env.jj_cmd_success(&repo_path, &["describe", "-m", "description 1"]);
+    let stdout = test_env.jj_cmd_success(
+        &repo_path,
+        &["log", "-T", "description if(divergent, \" !divergence!\")"],
+    );
+    // No divergence
+    insta::assert_snapshot!(stdout, @r###"
+    @ description 1
+    o (no description set)
+    "###);
+
+    // Create divergence
+    test_env.jj_cmd_success(
+        &repo_path,
+        &["describe", "-m", "description 2", "--at-operation", "@-"],
+    );
+    let stdout = test_env.jj_cmd_success(
+        &repo_path,
+        &["log", "-T", "description if(divergent, \" !divergence!\")"],
+    );
+    insta::assert_snapshot!(stdout, @r###"
+    Concurrent modification detected, resolving automatically.
+    o description 2
```

Comment:
@yuja @martinvonz Do you know why there is a line break here?

Reply:
I think that's simply because the description has a newline at the end. You can use `.first_line()` to avoid it.

Reply:
> You can use description.first_line() to avoid it.

I see, forgot the default template does that.